src/controllers/initgamecontroller.js

```diff
@@ -29,6 +29,10 @@ class InitGameController extends Controller
                 sim.registerEvent();
             }
         );
+
+        $(document).ready(function () {
```

Comment:
There should be no need for a `document ready`, as that has already happened. Just add the tutorial check in the `initGame()` function in `main.js`